sprocs/users_randomly_generate.sql

```diff
@@ -0,0 +1,60 @@
+
+    girl_names text[] := ARRAY['Emily', 'Emma', 'Madison', 'Abigail', 'Olivia', 'Isabella', 'Hannah', 'Samantha', 'Ava', 'Ashley', 'Sophia', 'Elizabeth', 'Alexis', 'Grace', 'Sarah', 'Alyssa', 'Mia', 'Natalie', 'Chloe', 'Brianna', 'Lauren', 'Ella', 'Anna', 'Taylor', 'Kayla', 'Hailey', 'Jessica', 'Victoria', 'Jasmine', 'Sydney', 'Julia', 'Destiny', 'Morgan', 'Kaitlyn', 'Savannah', 'Katherine', 'Alexandra', 'Rachel', 'Lily', 'Megan', 'Kaylee', 'Jennifer', 'Angelina', 'Makayla', 'Allison', 'Brooke', 'Maria', 'Trinity', 'Lillian', 'Mackenzie', 'Faith', 'Sofia', 'Riley', 'Haley', 'Gabrielle', 'Nicole', 'Kylie', 'Katelyn', 'Zoe', 'Paige', 'Gabriella', 'Jenna', 'Kimberly', 'Stephanie', 'Alexa', 'Avery', 'Andrea', 'Leah', 'Madeline', 'Nevaeh', 'Evelyn', 'Maya', 'Mary', 'Michelle', 'Jada', 'Sara', 'Audrey', 'Brooklyn', 'Vanessa', 'Amanda', 'Ariana', 'Rebecca', 'Caroline', 'Amelia', 'Mariah', 'Jordan', 'Jocelyn', 'Arianna', 'Isabel', 'Marissa', 'Autumn', 'Melanie', 'Aaliyah', 'Gracie', 'Claire', 'Isabelle', 'Molly', 'Mya', 'Diana', 'Katie', 'Leslie', 'Amber', 'Danielle', 'Melissa', 'Sierra', 'Madelyn', 'Addison', 'Bailey', 'Catherine', 'Gianna', 'Amy', 'Erin', 'Jade', 'Angela', 'Gabriela', 'Jacqueline', 'Shelby', 'Kennedy', 'Lydia', 'Alondra', 'Adriana', 'Daniela', 'Natalia', 'Breanna', 'Kathryn', 'Briana', 'Ashlyn', 'Rylee', 'Eva', 'Kendall', 'Peyton', 'Ruby', 'Alexandria', 'Sophie', 'Charlotte', 'Reagan', 'Valeria', 'Christina', 'Summer', 'Kate', 'Mikayla', 'Naomi', 'Layla', 'Miranda', 'Laura', 'Ana', 'Angel', 'Alicia', 'Daisy', 'Ciara', 'Margaret', 'Aubrey', 'Zoey', 'Skylar', 'Genesis', 'Payton', 'Courtney', 'Kylee', 'Kiara', 'Alexia', 'Jillian', 'Lindsey', 'Mckenzie', 'Karen', 'Giselle', 'Mariana', 'Valerie', 'Sabrina', 'Alana', 'Serenity', 'Kelsey', 'Cheyenne', 'Juliana', 'Lucy', 'Kelly', 'Sadie', 'Bianca', 'Kyra', 'Nadia', 'Lilly', 'Caitlyn', 'Jasmin', 'Ellie', 'Hope', 'Cassandra', 'Jazmin', 'Crystal', 'Jordyn', 'Cassidy', 'Delaney', 'Liliana', 'Angelica', 'Caitlin', 'Kyla', 'Jayla', 'Adrianna', 'Tiffany', 'Abby', 'Carly', 'Chelsea', 'Camila', 'Erica', 'Makenzie', 'Karla', 'Cadence', 'Paris', 'Veronica', 'Mckenna', 'Brenda', 'Bella', 'Maggie', 'Karina', 'Esmeralda', 'Erika', 'Makenna', 'Julianna', 'Elena', 'Mallory', 'Jamie', 'Alejandra', 'Cynthia', 'Ariel', 'Vivian', 'Jayden', 'Amaya', 'Dakota', 'Elise', 'Haylee', 'Josephine', 'Aniyah', 'Bethany', 'Keira', 'Aliyah', 'Laila', 'Camryn', 'Fatima', 'Reese', 'Annabelle', 'Monica', 'Lindsay', 'Kira', 'Selena', 'Macy', 'Hanna', 'Heaven', 'Clara', 'Katrina', 'Jazmine', 'Jadyn', 'Stella'];
+
+    first_names text[];
+
+    status int;
+    name text;
+    email text;
+    uin text;
```

Comment:
We don't seem to actually set the `uin`?

Reply:
I hesitated on what makes a good random UIN and then gave it up (because it's not important those accounts have one)? I guess we could make a hash or something -- I just didn't want them to conflict and trigger that workflow.

Reply:
Fair enough, then I'd delete this local variable.